### PR TITLE
Implement Noel

### PR DESCRIPTION
**Pull request by Rose-Personal**

### CODE REVIEW

File: data/card_definitions.json
Comment (Mach131) [inline data/card_definitions.json:6513]: It may be preferable to make this effect an "and" field under the above; that way it won't need to ask the player which one they want to resolve first (plus you won't need to repeat the timing and condition)

File: data/card_definitions.json
Comment (Mach131) [inline data/card_definitions.json:6747]: I think we usually put pass as the last choice, just for consistency

File: data/decks/noel.json
Comment (Mach131) [inline data/decks/noel.json:23]: make sure to add this as an exceed ability effect as well (we forgot in the existing s5 files, should be fixed soon)

File: data/decks/noel.json
Comment (Mach131) [inline data/decks/noel.json:31]: probably doesn't matter a huge amount, but it may be nice to line up the indentation here and in the exceed ability effects (unless that's just a github formatting thing?)

**Review by Mach131 — CHANGES_REQUESTED** [review]
Mostly looks good, just a few minor formatting things mostly~ Also, if you want, you can add a text effect description for "look_at_opponent_hand" in globals/card_definitions.gd; it probably doesn't matter since it's only used as a now effect, but might be nice for futureproofing

File: data/decks/noel.json
Comment (Rose-Personal) [reply data/decks/noel.json:31]: think this one is a github formatting thing, not seeing it in my own source files

**Review by Mach131 — APPROVED** [review]
looks good to me! will probably wait until DT sees this to merge (do you already have merge permissions? i'm not sure if i can give them myself)

**Comment by Rose-Personal** [comment]
I do not, sounds like a reasonable course of action to me!

File: data/card_definitions.json
Comment (daniel-k-taylor) [inline data/card_definitions.json:6571]: I know this is kind of annoying, but can you use the "reveal_hand" effect we already have an add an extra field that is "opponent": true?
Then just have an if statement in the "reveal_hand" handling code that says:
if 'opponent' in effect and effect['opponent']:
    opposing_player.reveal_hand()
else:
    performing_player.reveal_hand()

File: data/decks/noel.json
Comment (daniel-k-taylor) [inline data/decks/noel.json:31]: This is a tabs vs spaces thing.
Can you make it tabs to be consistent with the rest?

**Review by daniel-k-taylor — CHANGES_REQUESTED** [review]

**Review by daniel-k-taylor — APPROVED** [review]

**Pull request merged**
